components/theme/colors.js

```diff
@@ -24,150 +24,137 @@ colors.palette = palette(colors.OONI_BLUE, {
-  "blue7": "#0470a6",
-  "blue8": "#046191",
-  "blue9": "#034f77",
-  "indigo": "#0525cb",
-  "indigo0": "#f0f2ff",
-  "indigo1": "#d6dcfe",
-  "indigo2": "#b6c2fe",
-  "indigo3": "#92a3fc",
-  "indigo4": "#5e77fb",
-  "indigo5": "#0525cb",
-  "indigo6": "#0526d0",
-  "indigo7": "#0527d5",
-  "indigo8": "#0528db",
-  "indigo9": "#0528e0",
-  "violet": "#4805cb",
-  "violet0": "#f5f1ff",
-  "violet1": "#e5d7fe",
-  "violet2": "#d2bafe",
-  "violet3": "#b996fc",
-  "violet4": "#9664fb",
-  "violet5": "#4805cb",
-  "violet6": "#4a05cf",
-  "violet7": "#4b05d4",
-  "violet8": "#4d05d9",
-  "violet9": "#4e05dd",
-  "fuschia": "#ab05cb",
-  "fuschia0": "#fdeeff",
-  "fuschia1": "#f8d4fe",
-  "fuschia2": "#f2b5fd",
-  "fuschia3": "#eb8dfc",
-  "fuschia4": "#df50fa",
-  "fuschia5": "#ab05cb",
-  "fuschia6": "#a105bf",
-  "fuschia7": "#9705b4",
-  "fuschia8": "#8c04a6",
-  "fuschia9": "#800498",
-  "pink": "#cb0588",
-  "pink0": "#ffeefa",
-  "pink1": "#fed4ef",
-  "pink2": "#fdb5e5",
-  "pink3": "#fc8dd7",
-  "pink4": "#fa4ebf",
-  "pink5": "#cb0588",
-  "pink6": "#be0580",
-  "pink7": "#b10577",
-  "pink8": "#a2046d",
-  "pink9": "#910461",
-  "red": "#cb0525",
-  "red0": "#ffeff1",
-  "red1": "#fed6dc",
-  "red2": "#feb7c3",
-  "red3": "#fc91a2",
-  "red4": "#fa5770",
-  "red5": "#cb0525",
-  "red6": "#c00522",
-  "red7": "#b40521",
-  "red8": "#a7041d",
-  "red9": "#99041b",
-  "orange": "#cb4805",
-  "orange0": "#ffefe7",
-  "orange1": "#fed9c7",
-  "orange2": "#fcbfa0",
-  "orange3": "#fc9e6f",
-  "orange4": "#f96f2a",
-  "orange5": "#cb4805",
-  "orange6": "#bc4305",
-  "orange7": "#ac3d04",
-  "orange8": "#993704",
-  "orange9": "#842f04",
-  "yellow": "#cbab05",
-  "yellow0": "#fef4bf",
-  "yellow1": "#fce87b",
-  "yellow2": "#f9da32",
-  "yellow3": "#efca06",
-  "yellow4": "#debb05",
-  "yellow5": "#cbab05",
-  "yellow6": "#b69905",
-  "yellow7": "#9e8504",
-  "yellow8": "#826c04",
-  "yellow9": "#574a03",
-  "lime": "#88cb05",
-  "lime0": "#e1fdaa",
-  "lime1": "#c4fa59",
-  "lime2": "#a4f306",
-  "lime3": "#9be705",
-  "lime4": "#92d905",
-  "lime5": "#88cb05",
-  "lime6": "#7bb605",
-  "lime7": "#6a9d04",
-  "lime8": "#557f04",
-  "lime9": "#375203",
-  "green": "#25cb05",
-  "green0": "#d3fecb",
-  "green1": "#a6fc95",
-  "green2": "#57fa38",
-  "green3": "#2bed05",
-  "green4": "#28dd05",
-  "green5": "#25cb05",
-  "green6": "#21b605",
-  "green7": "#1c9f04",
-  "green8": "#168104",
-  "green9": "#0f5703",
-  "teal": "#05cb48",
-  "teal0": "#cdfede",
-  "teal1": "#99fcbb",
-  "teal2": "#3bfa7a",
-  "teal3": "#05ed54",
-  "teal4": "#05dd4e",
-  "teal5": "#05cb48",
-  "teal6": "#05b641",
-  "teal7": "#049e38",
-  "teal8": "#04812e",
-  "teal9": "#03571f",
-  "cyan": "#05cbab",
-  "cyan0": "#c3fef4",
-  "cyan1": "#84fce9",
-  "cyan2": "#06f7d2",
-  "cyan3": "#05eac6",
-  "cyan4": "#05dbb9",
-  "cyan5": "#05cbab",
-  "cyan6": "#05b699",
-  "cyan7": "#049e85",
-  "cyan8": "#04806b",
-  "cyan9": "#035548",
+  "black: "#000000",
```

Comment:
Are you sure black should be a solid black? Previously we were using a very dark shade of grey (I think similar to grey9)

Reply:
Yeah, I thought a bit about it and think so. Solid Black and White need to be present for UI things as they can be combined with opacity (which is also okay to do I'd say). For example, the test icons on the test cards are currently 20% black on their respective color. There is no color shade to achieve the same color. Pure white and black is best for opacity effects I'd say.

Reply:
As discussed on slack this should be the same hex code for `gray9` and `white` should be the same hex code for `gray0`